{"uuid": "94088d1c-9296-4e8e-abb9-1587ff303642", "children": ["4c48387b-782e-42bb-aad4-268b5dd4d25a"], "befores": [{"name": "setup_perfectoMobile_iOS", "status": "passed", "start": 1642660345763, "stop": 1642660369856}], "afters": [{"name": "setup_perfectoMobile_iOS::1", "status": "passed", "start": 1642660528557, "stop": 1642660528557}, {"name": "setup_perfectoMobile_iOS::teardown", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/TIP/wlan-testing/tests/e2e/interOp/conftest.py\", line 623, in teardown\n    driver.quit()\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/selenium/webdriver/remote/remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/selenium/webdriver/remote/remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/request.py\", line 74, in request\n    return self.request_encode_url(\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/request.py\", line 96, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/connectionpool.py\", line 699, in urlopen\n    httplib_response = self._make_request(\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/connectionpool.py\", line 382, in _make_request\n    self._validate_conn(conn)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/connectionpool.py\", line 1010, in _validate_conn\n    conn.connect()\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/connection.py\", line 358, in connect\n    conn = self._new_conn()\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/connection.py\", line 174, in _new_conn\n    conn = connection.create_connection(\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/urllib3/util/connection.py\", line 86, in create_connection\n    sock.connect(sa)\n"}, "start": 1642660528557, "stop": 1642660529300}], "start": 1642660345763, "stop": 1642660529301}